pages/linux/bluetoothd.md

```diff
@@ -0,0 +1,24 @@
+# bluetoothd
+
+> Daemon to manage bluetooth devices.
+> More information: <https://manned.org/bluetoothd>.
+
+- Start the daemon:
+
+`bluetoothd`
+
+- Start the daemon, logging to stdout:
+
+`bluetoothd --nodetach`
+
+- Start the daemon with a specific configuration file (defaults to `/etc/bluetooth/main.conf`):
+
+`bluetoothd --configfile {{path/to/file}}`
+
+- Start the daemon with verbose output to stderr:
+
+`bluetoothd --debug`
+
+- Start the daemon with verbose output coming from specific files in the bluetoothd source:
```

Comment:
Oh! I'm not sure if this only applies to source code from `bluetoothd` because this support plugins too, so:
```suggestion
- Start the daemon with verbose output coming from specific files in the source:
```

Reply:
Maybe
```suggestion
- Start the daemon with verbose output coming from specific files in the bluetoothd/plugins source:
```
?

Reply:
```suggestion
- Start the daemon with verbose output coming from specific files in the bluetoothd or plugins source:
```